astropy/modeling/core.py

```diff
@@ -801,6 +807,95 @@ def inverse(self):
         raise NotImplementedError("An analytical inverse transform has not "
                                   "been implemented for this model.")
 
+    @property
+    def bounding_box(self):
+        """
+        A `tuple` of length `n_inputs` defining the bounding box limits, or 
+        `None` for no bounding box. 
+
+        The default is `None`, or ``'auto'`` if ``bounding_box_default`` is defined 
+        in the model.
+
+        The ``bounding_box`` can be set manually to an array-like  object of shape 
+        ``(2 * model.n_inputs,)`` or ``(model.n_inputs, 2)``.
+
+        .. warning:: The limits are ordered according to the `numpy` indexing convention, 
+                    meaning that for inputs ``('x','y')`` the `bounding_box` limits are 
+                    defined as, ``((y_low, y_high),(x_low, x_high))``.
+
+        Examples
+        --------
+        >>> from astropy.modeling.models import Gaussian1D, Gaussian2D, custom_model
+        >>> model_1d = Gaussian1D()
+        >>> model_2d = Gaussian2D(x_stddev=1, y_stddev=1)
```

Comment:
Please use different values for x and y (e.g. `(1, 3)` instead of `(1, 1)`).
This will make it easier for users to understand the axis order used for bounding boxes if they use this example to learn how they work.
E.g. when I learn some new numpy, scipy, astropy, ... feature, I usually start by executing the examples in IPython and playing with them a bit, until I understand how it works.